### PR TITLE
Convert About page to Laravel

### PR DESCRIPTION
**Pull request by Jellyfrog**
Moved some stuff around also:
image

DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: app/Http/Controllers/AboutController.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            'callback_status' => $callback_status,
+            'callback_uuid'   => $callback_uuid,
+
+            'db_schema' => vsprintf('%s (%s)', $version->database()),
+            'git_log'   => $version->gitChangelog(),
+            'git_date'  => $version->gitDate(),
+            'project_name' => Config::get('project_name'),
+
+            'version_local'     => $version->local(),
+            'version_mysql'     => DB::select('select version()')[0]->{'version()'},
```
Comment (murrant) [inline app/Http/Controllers/AboutController.php]: Might through error if no response.
This might be better `current(DB::selectOne('select version()'))`

Comment (Jellyfrog) [reply app/Http/Controllers/AboutController.php]: Yes, way better.

File: resources/views/about/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+@section('javascript')
+<script>
+    $(document).ready(function () {
+        $("[name='statistics']").bootstrapSwitch('offColor','danger','size','mini');
+        $('input[name="statistics"]').on('switchChange.bootstrapSwitch',  function(event, state) {
+            event.preventDefault();
+            $.ajax({
+                type: 'POST',
+                url: '{{ url('/ajax/form') }}',
```
Comment (murrant) [inline resources/views/about/index.blade.php]: Just change this to `url: 'ajax_form.php',`
Looks like it was broken on accident last year...

File: resources/views/about/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+                error:function(){
+                    return $("#switch-state").bootstrapSwitch("toggle");
+                }
+            });
+        });
+        $('#clear-stats').click(function(event) {
+            event.preventDefault();
+            $.ajax({
+                type: 'POST',
+                url: '{{ url('/ajax/form') }}',
```
Comment (murrant) [inline resources/views/about/index.blade.php]: and here

**Comment by murrant** [comment]
Was looking at the callback stuff and made it work :)

File: resources/views/about/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,227 @@
...
+                    <td><b>@lang('RRDtool')</b></td>
+                    <td>{{ $version_rrdtool }}</td>
+                </tr>
+            </table>
+
+            <p>
+            <h3>@lang('LibreNMS is a community-based project')</h3>
+            @lang('Please feel free to join us and contribute code, documentation, and bug reports:')
+            <br>
+              <a href="http://www.librenms.org/">Web site</a> |
```
Comment (murrant) [inline resources/views/about/index.blade.php]: Translations missing for these links (Git log should probably be "Local git log" too)

File: resources/views/about/index.blade.php [hunk 5]
```diff
@@ -0,0 +1,227 @@
...
+              <a href="https://community.librenms.org/c/help">Bug tracker</a> |
+              <a href="https://community.librenms.org">Community Forum</a> |
+              <a href="http://twitter.com/librenms">Twitter</a> |
+              <a href="http://www.librenms.org/changelog.html">Changelog</a> |
+              <a href="#" data-toggle="modal" data-target="#git_log">Git log</a>
+          </p>
+
+          <h3>@lang('Contributors')</h3>
+
+          <p>See the <a href="https://github.com/librenms/librenms/blob/master/AUTHORS.md">list of contributors</a> on GitHub.</p>
```
Comment (murrant) [inline resources/views/about/index.blade.php]: Probably like this:
```
@lang('See the <a href=":url">list of contributors</a> on GitHub.', ['url' => 'https://github.com/librenms/librenms/blob/master/AUTHORS.md'])
```

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-55-release-changelog-august-2019/9428/1

**Pull request merged**